### ISSUE TITLE
On reproducing the results.

### ISSUE THREAD

**Issue by mariolew** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#83]
Hello, I can't get close results with the "trained from scratch model" with train_VGG.py.  And in the training process, the validation loss is always 0.0, and I haven't found the reason. BTW, the names of the mask files are different with those in your code, but the code can be run by changing the names.

**Comment by FoolishMao** [comment]
what your loss curve looks like? My is a bit wired!! train loss is very little,and validation loss is always 0.0.

**Comment by kentaroy47** [comment]
@mariolew @FoolishMao 
did you solve this? I also get validation loss 0.0 and low train loss from the beginning as well..
